Update GStreamer to latest 1.26.4 release

thiblahute opened the pull request.

thiblahute committed:
Update GStreamer to latest 1.26.4 release

thiblahute committed:
Update python dependencies

hfiguiere commented:
maybe at that point updating to GNOME 48 would be a good opportunity.

thiblahute commented:
@hfiguiere You [already did it](https://github.com/flathub/org.pitivi.Pitivi/pull/47) afaict, I am updating all the other deps.

hfiguiere commented:
duh. I checked earlier, not sure what I saw.

Sorry for the noise.

(yes python dependencies are kind of a pain)

thiblahute committed:
Remove LLVM dependency to fix build

- Remove llvmlite and numba from Python modules as LLVM 14 is not available for SDK 24.08
- Remove LLVM SDK extension requirement
- librosa will work without numba (with reduced performance)
- This fixes the build failure caused by missing org.freedesktop.Sdk.Extension.llvm14

thiblahute committed:
Fix librosa build: use wheels for numba/llvmlite to avoid LLVM requirement

- Replace numba and llvmlite source distributions with pre-built wheels for x86_64 and aarch64
- This avoids the need for LLVM 14 which is not available in SDK 24.08
- Update script to automatically fetch wheels for both architectures
- Remove duplicate source entries from generated JSON
- This fixes the CI build failures related to missing audioread and LLVM compatibility

thiblahute commented:
This is so messy, basically in the end we have a dependency on LLVM14 for `librosa`, which we would need to build (the extension for the GNOM 48 SDK is LLVM18, no way to use LLVM14), but there is no way I build llvm here ! I am tempted to just use wheels, that would be a way forward, but I dislike that idea :|
